pom.xml

```diff
@@ -550,6 +550,7 @@
         <version.org.jboss.metadata>16.0.0.Final</version.org.jboss.metadata>
         <version.org.jboss.mod_cluster>2.0.4.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>7.1.0.Final</version.org.jboss.narayana>
+        <version.org.jboss.narayana.lra>0.0.9.Final</version.org.jboss.narayana.lra>
```

Comment:
Does the 0.0.9 version imply anything about stability? If it does, will it be at a stable version in time to be upgrade in WF by December 18?

Reply:
@bstansberry no it doesn't. Marco told me he just chose this version because he wants to do the 1.0.0 as a normal upgrade once integrated to go through all verifications. The code is forked from narayana main and it's the same.